### ISSUE TITLE
Can't dispatch to sensor : ServiceDNSName wrong assumption

### ISSUE THREAD

**Issue by fayak** [argoproj/argo-events#443]
**Describe the bug**

When dispatching an event to a sensor, the gateway implies that the cluster name is `cluster.local` by sending it to `<sensor-name>.svc.cluster.local` (common/util.go:40).

However, if a Kubernetes cluster (such ours) has a different name (like example.com), this assumption leads to an unreachable sensor.

A fix for that would be:
```
common/util.go:40
func ServiceDNSName(serviceName, namespace string) string {
-       return fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, namespace)
+       return fmt.Sprintf("%s.%s.svc", serviceName, namespace)
```
Another one would be to add a config key for that maybe ?

**Comment by VaibhavPage** [comment]
This is closely related to https://github.com/argoproj/argo-events/issues/341#issuecomment-554356170

**Comment by VaibhavPage** [comment]
I have put in the fix for v0.12
